src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -354,7 +354,6 @@ class NBDServer
     void* entry() override
     {
       (server.*func)();
-      server.shutdown();
```

Comment:
@trociny seems I haven't completely understand this flow. How about I dropped this commit and only leave the bug fix: "remove debug messages from do_unmap" ?

I will think carefully and pulled in future if necessary.